mne/viz/topomap.py

```diff
@@ -418,10 +405,10 @@ def plot_topomap(data, pos, vmax=None, vmin=None, cmap='RdBu_r', sensors=True,
         xlim = np.inf, -np.inf,
         ylim = np.inf, -np.inf,
         mask_ = np.c_[outlines['mask_pos']]
-        xmin, xmax = (np.min(np.r_[xlim[0], mask_[:, 0] * 1.01]),
-                      np.max(np.r_[xlim[1], mask_[:, 0] * 1.01]))
-        ymin, ymax = (np.min(np.r_[ylim[0], mask_[:, 1] * 1.01]),
-                      np.max(np.r_[ylim[1], mask_[:, 1] * 1.01]))
+        xmin, xmax = (np.min(np.r_[xlim[0], mask_[:, 0]]),
+                      np.max(np.r_[xlim[1], mask_[:, 0]]))
+        ymin, ymax = (np.min(np.r_[ylim[0], mask_[:, 1]]),
+                      np.max(np.r_[ylim[1], mask_[:, 1]]))
```

Comment:
BTW it would be more efficient to do a min of two mins than a min of a concat, but it doesn't really matter here